cats/cats-redis/src/test/groovy/com/netflix/spinnaker/cats/redis/cluster/DefaultNodeIdentitySpec.groovy

```diff
@@ -16,8 +16,11 @@
 
 package com.netflix.spinnaker.cats.redis.cluster
 
+import com.netflix.spinnaker.cats.redis.test.NetworkUnavailableCheck
+import spock.lang.IgnoreIf
 import spock.lang.Specification
 
+@IgnoreIf({NetworkUnavailableCheck.networkUnavailable()})
```

Comment:
unrelated to this PR I just happened to discover this test doesn't like it if you are trying to run with no network connection